addons/debugger/userscript.js

```diff
@@ -97,6 +108,11 @@ export default async function ({ addon, global, console, msg }) {
     myFlash.block = block;
 
     function _flash() {
+      if (!myFlash.block.svgPath_) {
+        myFlash.timerID = count = 0;
+        flashOn = true;
+        return;
+      }
```

Comment:
This code is taken from devtools. is it worth bringing this change there?

Reply:
Gonna check that, bc that means devtools can crash...

Reply:
Cannot repro crash, so not gonna fix in this PR.